src/SLCore/Common/Helpers/ConnectionIdHelper.cs

```diff
@@ -33,6 +35,22 @@ internal class ConnectionIdHelper : IConnectionIdHelper
         private const string SonarQubePrefix = "sq|";
         private static readonly Uri SonarCloudUri = new Uri("https://sonarcloud.io");
 
+        public string GetConnectionIdFromUri(Uri uri, string organisation)
+        {
+            if (uri is not null)
+            {
+                if (uri == SonarCloudUri)
+                {
+                    if (!string.IsNullOrWhiteSpace(organisation)) { return SonarCloudPrefix + organisation; }
```

Comment:
newlines missing in if body

Reply:
it's to reduce clutter I think the expression is simple enough to read easily.

Reply:
I think you're treating a symptom here. This method is hard to read, I completely missed the branch where organization is null (is that even a possibility?)